src/planner/planner.cpp

```diff
@@ -52,6 +52,19 @@ void Planner::CreatePlan(SQLStatement &statement) {
 		this->plan = nullptr;
 		parameters_resolved = false;
 	} catch (const Exception &ex) {
+		auto &config = DBConfig::GetConfig(context);
+
+		for (auto &extension_op : config.operator_extensions) {
+			auto bound_statement =
+			    extension_op.Bind(context, *this->binder, extension_op.operator_info.get(), statement);
+			if (bound_statement.plan != nullptr) {
+				this->names = bound_statement.names;
+				this->types = bound_statement.types;
+				this->plan = move(bound_statement.plan);
+				break;
```

Comment:
Won't this still throw the exception - as this break only breaks out of the loop and into the `throw` statement?

Reply:
Thanks for taking close look at my PR. Indeed I staged my commit incorrectly and forgot the guard check. Doing another round of testing and will push the update.